src/schema/artist/related.ts

```diff
@@ -68,7 +68,7 @@ export const Related = {
               : relatedContemporaryArtistsLoader
 
           return fetch(gravityArgs).then(({ body, headers }) => {
-            const totalCount = headers["x-total-count"]
+            const totalCount = parseInt(headers["x-total-count"] || "0", 10)
```

Comment:
Any particular reason we are being so explicit as to provide the radix here?

Reply:
The TS types make it a required parameter, because of:

> If the input `string` begins with "0", radix is eight (octal) or 10 (decimal).  Exactly which radix is chosen is implementation-dependent.  ECMAScript 5 specifies that 10 (decimal) is used, but not all browsers support this yet.  For this reason **always specify a radix when using parseInt**.

– https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt

Reply:
Oh snap! Good to know.

Reply:
today I learned, I just accepted it and moved on - glad you asked